.github/actionlint.yaml

```diff
@@ -1,4 +1,4 @@
 self-hosted-runner:
   # Labels of self-hosted runners in array of string
   labels:
-    - high-perf-docker
+    - runs-on,cpu=64,family=c7,hdd=500,image=ubuntu22-full-x64,run-id=${{ github.run_id }}
```

Comment:
Is this necessary ? It looks like a label not a runner spec.

Reply:
this is just to tell the actionlint linter not to complain about this in the action yamls.